crates/cast/tests/data/scripts/hello_world/Scarb.toml

```
@@ -0,0 +1,6 @@
+[package]
+name = "hello_world"
+version = "0.1.0"
+
+[dependencies]
+starknet = "2.3.0"
```

```suggestion
starknet = ">=2.3.0"
```

to avoid having to bump this each time we bump scarb version?